Helper/Order.php

```diff
@@ -712,12 +726,29 @@ public function saveUpdateOrder($reference, $storeId = null, $boltTraceId = null
                     if ($transaction->status == Payment::TRANSACTION_AUTHORIZED) {
                         $this->voidTransactionOnBolt($transaction->id, $storeId);
                     }
-                    $this->bugsnag->notifyException($exception);
-                    return $this;
+
+                ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+                //// Allow missing quote failed hooks to resend 10 times so the Administrator can be notified via email
+                ///  On the eleventh time that the failed hook was sent to Magento, we return $this in order to have the hook return successfully.
+                ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+                    /** @var \Bolt\Boltpay\Model\Log $logFactory */
+                    $logFactory = $this->logFactory->create();
```

Comment:
Let's put this behind a feature switch with default on

Reply:
> Let's put this behind a feature switch with default on

@roopakv  I traced the code and saw this http://prntscr.com/r4o044
Looks like I will need to do the same but I'm not sure how feature switch works. Can you give me a detailed document about how to use feature switch?

Reply:
@john sure I can do so in 10 minutes. If you look at the PR for the line of code you pointed to, that should be it. It's pretty light weight